src/renderer/features/proxies/workers/proxy-worker.ts

```diff
@@ -228,6 +232,21 @@ async function getProxies({
   });
   console.log(`proxy-worker ${api.genesisHash}: 🟣 proxied accounts to remove: `, proxiedAccountsToRemove);
 
+  if (proxyUrl && proxiedAccountsToAdd.length) {
```

Comment:
This should not be done inside the worker
Much more cleaner way is to call QGL inside `getProxiesFx` after we receive data from `endpoint.call.getProxies`
enrich `proxiedAccountsToAdd` if needed and return the data